### PR TITLE
Create Add-SPOSiteScriptPackage.md

### PR DESCRIPTION
**Pull request by alinsirbu**

### CODE REVIEW

**Comment by alinsirbu** [comment]
@microsoft-github-policy-service agree [company="{Microsoft}"]

**Comment by alinsirbu** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Add-SPOSiteScriptPackage -Title <String> -ContentPath <String> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Uploads a new site script package for use either directly or in a site design. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## EXAMPLES
+
+### Example
+This example adds a site script package as a zip file containing a manifest.json script, as well as a Dataverse solution zip file with a Power Automate flow definition. The content of manifest.json is as follows:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md:25]: please move the description at the end of the example

Comment (alinsirbu) [reply sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md]: @get-itips Done, thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+```
+Add-SPOSiteScriptPackage -Title <String> -ContentPath <String> [-Description <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Uploads a new site script package for use either directly or in a site design. The package file must be a zip archive containing all the files necessary for the site script. A file called “manifest.json” with script actions must be present in this zip file.
+
+## EXAMPLES
+
+### Example
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md:24]:
```suggestion
### Example 1
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+$compress = @{
+  Path = ".\manifest.json", ".\solution.zip"
+  DestinationPath = "c:\scripts\site-script-package.zip"
+}
+Compress-Archive @compress
+
+Add-SPOSiteScriptPackage -Title "Install Contoso flow" -Description "Installs the new Contoso flow in a list" -ContentPath "c:\scripts\site-script-package.zip"
+```
+
+This example adds a site script package as a zip file containing a manifest.json with script actions as exemplified above, as well as a Dataverse solution zip file with a Power Automate flow definition.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md:48]:
```suggestion
This example adds a site script package as a zip file containing a manifest.json with script actions as exemplified previously, as well as a Dataverse solution zip file with a Power Automate flow definition.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### System.Object
+## NOTES
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md:107]:
```suggestion

## NOTES
```

File: sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Add-SPOSiteScriptPackage.md:98]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @alinsirbu, one more, please.

**Comment by alinsirbu** [comment]
@tseward you're listed here as a pending reviewer
"tseward was requested for review"

**Comment by alinsirbu** [comment]
Looking for approvals from @JohanFreelancer9 and @get-itips

**Review by get-itips — APPROVED** [review]

**Pull request merged**